spec/lib/mvi/service_spec.rb

```diff
@@ -321,4 +321,30 @@
       end
     end
   end
+
+  describe '.find_profile monitoring' do
+    context 'with a successful request' do
+      it 'should increment find_profile total' do
+        allow(user).to receive(:mhv_icn).and_return('1008714701V416111^NI^200M^USVHA^P')
```

Comment:
Is the string that's returned meaningful? It might be more readable if it's stored in a well named variable, or if we can remove `and_return` altogether.

Reply:
It's totally unnecessary, good call.